cpu/cpu_linux.go

```diff
@@ -55,19 +56,36 @@ func CPUTimes(percpu bool) ([]CPUTimesStat, error) {
 	return ret, nil
 }
 
+func finishCPUInfo(c *CPUInfoStat) error {
+	if c.Mhz == 0 {
+		lines, err := common.ReadLines(fmt.Sprintf("/sys/devices/system/cpu/cpu%d/cpufreq/cpuinfo_max_freq", c.CPU))
+		if err == nil {
+			value, err := strconv.ParseFloat(lines[0], 64)
+			if err != nil {
+				return err
+			}
+			c.Mhz = value
+		}
+	}
+	if len(c.CoreID) == 0 {
+		lines, err := common.ReadLines(fmt.Sprintf("/sys/devices/system/cpu/cpu%d/topology/core_id", c.CPU))
+		if err == nil {	
+			c.CoreID = lines[0]
+		}
+	}
+	return nil
+}
+
 func CPUInfo() ([]CPUInfoStat, error) {
 	filename := "/proc/cpuinfo"
```

Comment:
Sorry, I merged another PR after your PR is sent. 
Could you change to use `common.GetEnv`  like https://github.com/shirou/gopsutil/blob/master/cpu/cpu_linux.go#L59 ? (See #97 )